Dockerfile

```diff
@@ -12,7 +12,11 @@ ARG PORT_DEBUG
 ENV PORT ${PORT}
 EXPOSE ${PORT} ${PORT_DEBUG}
 
+RUN mkdir -p /home/node/app/node_modules && chown -R node:node /home/node/app
```

Comment:
Why we have to create node_modules, it should be automatically created?

Reply:
this is to ensure the folder have only node permission ,however I do change user to node before  nom install so you right we should't need this ,  thanks good point,,